src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -1606,4 +1607,43 @@ private static boolean isGVCFReferenceBlock( final VariantContext vc ) {
                vc.getAlternateAlleles().size() == 1 &&
                vc.getAlternateAllele(0).equals(Allele.NON_REF_ALLELE);
     }
+
+
+    @DataProvider(name="PairHMMResultsModes")
+    public Object[][] PairHMMResultsModes() {
+        return new Object[][] {
+                //{PairHMM.Implementation.AVX_LOGLESS_CACHING, new File(TEST_FILES_DIR, "expected.AVX.hmmresults.txt")}, //TODO disabled because of failures on travis
```

Comment:
Can you elaborate on why this is disabled? Ideally we want test coverage for the default HMM in GATK...